Replace rand() with SecureRandom.random_number()

dusk0r opened the pull request.

Commit:
Replace rand() with SecureRandom.random_number()

nricciar commented:
I wouldn’t mind merging this, but I would prefer it done in a backwards compatible way as SecureRandom does not look like its included with Ruby 1.8.x.  So if this patch could fall back on rand if SecureRandom does not exist I would be happy to merge it.

If you are unable to make those changes I will probably have some time to do it sooner or later.

dusk0r committed:
Backward compatibility for SecureRandom

Ruby >= 1.8.7 should support SecureRandom
http://www.rubyinside.com/ruby-187-released-912.html

dusk0r commented:
Ok, i added a fix. Is this ok?

nricciar commented:
looks good
